install.md

```diff
@@ -97,17 +73,8 @@ sudo apt-get -y update
 sudo apt-get -y install skopeo
 ```
 
-If you would prefer newer (though not as well-tested) packages,
-the [Kubic project](https://build.opensuse.org/package/show/devel:kubic:libcontainers:stable/skopeo)
-provides packages for active Ubuntu releases 20.04 and newer (it should also work with direct derivatives like Pop!\_OS).
-Checkout the [Kubic project page](https://build.opensuse.org/package/show/devel:kubic:libcontainers:stable/skopeo)
-for a list of supported Ubuntu version and
-architecture combinations. **NOTE:** The command `sudo apt-get -y upgrade`
-maybe required in some cases if Skopeo cannot be installed without it.
-The build sources for the Kubic packages can be found [here](https://gitlab.com/rhcontainerbot/skopeo/-/tree/debian/debian).
-
-CAUTION: On Ubuntu 20.10 and newer, we highly recommend you use Buildah, Podman and Skopeo ONLY from EITHER the Kubic repo
-OR the official Ubuntu repos. Mixing and matching may lead to unpredictable situations including installation conflicts.
+The [Kubic project](https://build.opensuse.org/package/show/devel:kubic:libcontainers:stable/skopeo)
+provides packages for active Ubuntu 20.04 (it should also work with direct derivatives like Pop!\_OS).
```

Comment:
```suggestion
provides packages for Ubuntu 20.04 (it should also work with direct derivatives like Pop!\_OS).
```

Not sure what "active" means.

Reply:
is it just 20.24 or "20.24 and later"?

Reply:
> Not sure what "active" means.

@vrothberg ha, `active` was a poor choice, I wanted to mean something that's not EOL, and yes, we could just get rid of that word.



> is it just 20.24 or "20.24 and later"?

@TomSweeneyRedHat in the case of podman, we're not mentioning kubic repos if the package exists in the default distro repos. I'm hoping to do the same here.